.codespell.exclude

```diff
@@ -8,3 +8,6 @@ NotIn
 groupin
 showIn
 Bein
+prexisting
+intoto
+InToto
```

Comment:
Any chance we can ignore these by ignoring the Kyverno CRD entirely from the spellcheck?